SamplesNative/SamplesNative.vcxproj

```diff
@@ -220,7 +221,18 @@
     </ClCompile>
     <ClCompile Include="DeviceLostHelper.cpp" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Any changes in SamplesNative are surprising to me. Are they intentional? Especially weird is that native samples depend on .NETCore. Pretty sure you don't want any of the changes in this directory